theano/sandbox/cuda/opt_util.py

```diff
@@ -73,8 +73,21 @@ def opt(node):
                 if lr is None or targ is None:
                     return None
                 inputs = list(targ.inputs)
-                inputs[alpha_in] = lr * targ.inputs[alpha_in]
-                inputs[beta_in] = lr * targ.inputs[beta_in]
+                try:
+                    raise NotScalarConstantError()
```

Comment:
Is that supposed to be the final version? The code below will always be skipped.